README.md

```diff
@@ -2,31 +2,345 @@
+considered an access modifier. In this lesson, that is the only access modifier
+we will look at. We'll also use it in conjunction with the `static` keyword.
+Again, more on those keywords later on.
+
+### Return Values
+
+A method can return a value after it's done executing. In Java, we want to
+specify what data type is to be returned once the method has completed. We call
+this a **return type**. It may return another object, a primitive, or even
+nothing!
+
+Consider the `println()` method for example. The job of the `println()` method is
+to print text to the console. Once it's done performing that work, it simply
+returns the execution to the method that called it without returning any value.
+This is expressed in the header of the method with the `void` keyword:
+
+```java
+public void thisMethodReturnsNothing() {
+    System.out.println("I print something on the screen, and then return nothing");     
+} 
+```
+
+Another example is the `public static void main(String[] args)` method we have
+been referring to. This method also makes use of the `void` return type to tell
+us that it will not have a return value when the program finishes executing.
+
+Here are a few rules to remember about method return types:
+
+- A method must always have a return type, but that type can be `void` to
+  specify that the method does not return anything.
+- A method return type can be any valid Java type, which means it can be any
+  pre-defined Java type or any of our own classes we create. More on that point
+  later.
+- If a method is declared with a return type other than `void`, it _must_ return
+  a value compatible with that type.
+
+### Method Name
+
+The **method name** is what we want to call our method. When naming methods,
+it is best to be descriptive yet concise with the action that is to be performed
+inside the method. For example, if we expect the method to cook us some food, we
+should not name the method `x()` but rather `cook()`.
+
+Simplifying the method name, yet being descriptive with the name to make it clear
+to other developers is very helpful.
+
+Method names should also follow the same naming conventions of variables. In
+Java, the name of the method usually starts with a lowercase character. If the
+name of the method has multiple words in it, the first word will start with a
+lowercase character and every word after will start with an uppercase character.
+For example: `cookEggs()`. This naming convention is referred to as
+**camel case**.
+
+### Parameter List
+
+A **parameter list** is a list of the declarations needed for the method to
+function properly. Each parameter in the parameter list needs to be in a
+declaration format as such: `<type> <variableName>`. It is also possible to not
+have a parameter list. An empty parameter list would look like this: `()`. If
+multiple parameters exist, then they are to be separated by a comma `,` between
+each parameter. Example: `cook("eggs", "bacon")`.
```

Comment:
Should this be `cook(String item1, String item2)`? It sounds like this paragraph is more about how to write a method rather than call a method.